src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-credentialId.html

```diff
@@ -1 +1 @@
-<div>CodeSonar credentials</div>
\ No newline at end of file
+<div>The hub user account credentials, they can be provided with a Jenkins Credential parameter of type <i>Username with password</i>, or with a Credential parameter of type of type <i>Certificate</i>, specified in the PKCS#12 format.</div>
```

Comment:
Hub user account credentials to authenticate the CodeSonar operations that you will be invoking within Jenkins. Provide one of the following:
- A Credentials parameter of type **Username with password**, containing the hub user account username and password.
- A Credentials parameter of type **Certificate**, containing the user certificate and its private key in PKCS#12 format.





 This account must have sufficient permissions to:

- Sign into the hub.
- Analyze the project you are interested in.
- Create a launch daemon.
- Browse analysis results.

With factory settings, it is sufficient for the hub user account to have the built-in `Enabled` and `User` roles.

Reply:
I think it is best to make the very first sentence into a very brief summary description.  So I suggest adjusting the first sentences as follows:

> CodeSonar Hub user account credentials.
>
> Credentials used to authenticate the CodeSonar operations that you will be invoking within Jenkins.
> Provide one of the following...

Otherwise, I think this description looks much much better than what we had.  Thanks!